charts/seed-controlplane/charts/etcd/templates/etcd-backuprestore-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Do we really need three separate services targeting the same pod? Can't we just re-use the currently existing service and open up the other ports as well?

Reply:
As Rafael suggested, me too agree on having single file.  Let the service instance name also be as it is. Otherwise we might have to change it at kube-apiserver and tls certificate configurations as well.

Reply:
Done